pkgs/applications/graphics/freecad/default.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, mkDerivation, fetchurl, cmake, ninja, coin3d, xercesc, ode, eigen, qt5, opencascade-occt, gts
-, hdf5, vtk, medfile, zlib, python3Packages, swig, gfortran, libXmu
-, soqt, libf2c, libGLU, makeWrapper, pkgconfig
+{ stdenv, mkDerivation, fetchurl, fetchpatch, cmake, ninja, coin3d, xercesc, ode
+, eigen, qt5, opencascade-occt, gts, hdf5, vtk, medfile, zlib, python3Packages
```

Comment:
This expression shouldn't reference `qt5` and instead reference all the attributes within that set directly as they're in scope.

Reply:
Right you are. Updated.

Reply:
This is what I meant precisely https://github.com/NixOS/nixpkgs/issues/65399#issuecomment-520915488

Reply:
Oops, sorry. When I was hunting around for info I only happened to see bad examples. Updated again.